apps/expo/src/app/index.tsx

```diff
@@ -97,31 +96,37 @@ function CreatePost() {
 }
 
 export default function Index() {
+  const { setColorScheme } = useColorScheme();
   const utils = api.useUtils();
 
   const postQuery = api.post.all.useQuery();
 
   const deletePostMutation = api.post.delete.useMutation({
-    onSettled: () => utils.post.all.invalidate(),
+    onSettled: () => utils.post.all.invalidate().then(),
   });
 
+  useEffect(() => {
```

Comment:
Can't this be in a root / set in some provider instead of an effect?

Reply:
Removed it entirely as it is not needed